integrations/tensorflow/python_projects/iree_tflite/iree/tools/tflite/scripts/iree_import_tflite/__main__.py

```diff
@@ -4,18 +4,58 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-import subprocess
+import argparse
+import inspect
+from pathlib import Path
+import re
 import sys
-
 import iree.tools.tflite
 
 
-def main(args=None):
-  if args is None:
-    args = sys.argv[1:]
-  exe = iree.tools.tflite.get_tool("iree-import-tflite")
-  return subprocess.call(args=[exe] + args)
```

Comment:
We have benchmarking infrastructure passing an unsupported flag to this reworked CLI: https://github.com/openxla/iree/actions/runs/4862317331/jobs/8668737295?pr=13361#step:7:351

```
FAILED: e2e_test_artifacts/iree_PersonDetect_int8_93c0f75188363af77647cb2f1deb41446575c9cd084d86119287156eb181d850.mlir /work/build-e2e-test-artifacts/e2e_test_artifacts/iree_PersonDetect_int8_93c0f75188363af77647cb2f1deb41446575c9cd084d86119287156eb181d850.mlir 
cd /work/build-e2e-test-artifacts/tests/e2e/test_artifacts && /home/runner/.local/bin/iree-import-tflite /work/build-e2e-test-artifacts/e2e_test_artifacts/model_bc1338be-e3df-44fd-82e4-40ba9560a073_PersonDetect_int8.tflite -o=/work/build-e2e-test-artifacts/e2e_test_artifacts/iree_PersonDetect_int8_93c0f75188363af77647cb2f1deb41446575c9cd084d86119287156eb181d850.mlir --output-format=mlir-bytecode
usage: iree-import-tflite [-h] -o OUTPUT_PATH [--input-array INPUT_ARRAYS]
                          [--output-array OUTPUT_ARRAYS]
                          flatbuffer
iree-import-tflite: error: unrecognized arguments: --output-format=mlir-bytecode
```

source (some of this is generated):
https://github.com/openxla/iree/blob/0c0c34f7c8d5a920942f888db4521f64737d598c/tests/e2e/test_artifacts/generated_e2e_test_iree_artifacts.cmake#L1-L9 https://github.com/openxla/iree/blob/0c0c34f7c8d5a920942f888db4521f64737d598c/build_tools/cmake/iree_benchmark_suite.cmake#L41-L50

Reply:
The flag in `generated_e2e_test_iree_artifacts.cmake` comes from https://github.com/openxla/iree/blob/main/build_tools/python/e2e_test_framework/definitions/iree_definitions.py#L194